rst_files/wald_friedman.rst

```diff
@@ -401,467 +345,201 @@ that appear on the right side of the Bellman equation as separate functions
 
 Later, doing this will help us obey **the don't repeat yourself (DRY)** golden rule of coding
 
-
 Implementation
 ==================
 
-Let's code this problem up and solve it
-
-To approximate the value function that solves Bellman equation :eq:`new1`, we
-use value function iteration
-
-* For earlier examples of this technique see the :doc:`shortest path <short_path>`, :doc:`job search <mccall_model>` or :doc:`optimal growth <optgrowth>` lectures
-
-As in the :doc:`optimal growth lecture <optgrowth>`, to approximate a continuous value function
-
-* We iterate at a finite grid of possible values of :math:`p`
-
-* When we evaluate :math:`A(p)` between grid points, we use linear interpolation
-
-This means that to evaluate :math:`J(p)` where :math:`p` is not a grid point, we must use two points:
-
-* First, we use the largest of all the grid points smaller than :math:`p`, and call it :math:`p_i`
-
-* Second, we use the grid point immediately after :math:`p`, named :math:`p_{i+1}`, to approximate the function value as
+Let's code this problem up and solve it.
 
-.. math::
-
-    J(p) = J(p_i) + (p - p_i) \frac{J(p_{i+1}) - J(p_i)}{p_{i+1} - p_{i}}
+We implement the cost functions for each choice considered in the
```

Comment:
This is a break from writing the style adopted in most of the lectures.

In most places on lectures.quantecon.org we have one sentence paragraphs without punctuation at the end. Here we have replaced that with more conventional paragraphs that have multiple sentences separated by punctuation marks.

I'm not casting a vote here for which I think we should have going forward, my purpose is only to point out that the change occurred

This applies to a few places in the diff, but I'll only mention it once

Reply:
Right.  Let's stick to one sentence lines with a line break after and no full stop.

That's something we can change later but let's at least be uniform.

Reply:
Good catch. I have modified it to be consistent with that part of the style.